core/domain/suggestion_services.py

```diff
@@ -902,6 +902,29 @@ def auto_reject_translation_suggestions_for_exp_ids(exp_ids: List[str]) -> None:
         suggestion_models.INVALID_STORY_REJECT_TRANSLATION_SUGGESTIONS_MSG)
 
 
+def auto_reject_translation_suggestions_for_content_ids(
```

Comment:
Can we rather name this 
```suggestion
def reject_translation_suggestions_for_content_ids(
```

I don't see why we need the `auto` we still have to run this manually. This might be also updated for the other functions like this.

Reply:
Future lesson content changes that make translation suggestions obsolete will automatically trigger rejections. I think the "auto" part is mostly to signal that the suggestion reviewer is set to OppiaSuggestionBot and to differentiate it from a normal rejection from a human user.

Reply:
Hmm, the "auto" prefix doesn't do that for me, but on the other hand I do not have any suggestion for a different naming.

Reply:
Ack.